build/commands/lib/util.js

```diff
@@ -638,7 +638,7 @@ const util = {
     // base args
     const initialArgs = ['sync', '--reset', '--nohooks']
     const chromiumArgs = ['--revision', 'src@' + config.getProjectRef('chrome')]
-    const resetArgs = ['--with_tags', '--with_branch_heads', '--upstream']
```

Comment:
don't we refer to chromium versions as tags and won't this remove fetching them?

Reply:
gclient has an ability to fetch unknown commit . But let me check twice;)

Reply:
But this PR builds even with `TERMINATE_NODE` which should wipe gitcache

Reply:
Unfortunately I'm not sure that removing tags will not break checkout in some corner cases. gclient inside has a obscure behaviour with a lot of cases.
But `with_branch_heads` can be safely removed, lets do it step by step.

Reply:
@bridiver "accidentally" removed `with_branch_heads` in April 2020 then added it back 11 days later ([associated commit](https://github.com/brave/brave-core/commit/cd2bef907dd5e3aca2e09c5d64615e8eafc65bef)). I couldn't find a discussion of the flag in the Slack channels I have access to, nor here on GitHub. But the fact that it was added back 11 days after being removed makes me think there might be a reason why it's there?

Reply:
I have to agree that it is simple to leave with_branch_heads here than get a build issue in the future.
I've found that 3rd parties also checkouts with this flag and have these branches.

So, let's decline this PR